### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by synicalsyntax**
#24

### CODE REVIEW

**Comment by zulipbot-test** [comment]
Hello @zulipbot-testing/server-user-docs members, this pull request needs your attention!

**Comment by zulipbot-test** [comment]
Hello @zulipbot-testing/server-emoji members, this pull request needs your attention!

**Comment by synicalsyntax** [comment]
Reference issue #24

**Comment by zulipbot-test** [comment]
Hello @zulipbot-testing/server-user-docs members, this pull request needs your attention!

**Comment by zulipbot-test** [comment]
Hello @zulipbot-testing/server-emoji members, this pull request needs your attention!

**Comment by synicalsyntax** [comment]
#24

**Comment by zulipbot-test** [comment]
Hello @zulipbot-testing/server-emoji members, this pull request needs your attention!

**Comment by zulipbot-test** [comment]
Hello @zulipbot-testing/server-user-docs members, this pull request needs your attention!

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# test
```
Comment (synicalsyntax) [inline README.md]: #24

**Comment by zulipbot-test** [comment]
Hello @zulipbot-testing/server-emoji members, this pull request needs your attention!

**Comment by zulipbot-test** [comment]
Hello @zulipbot-testing/server-user-docs members, this pull request needs your attention!

**Comment by zulipbot-test** [comment]
Hello @zulipbot-testing/server-user-docs members, this pull request needs your attention!

**Comment by zulipbot-test** [comment]
Hello @zulipbot-testing/server-emoji members, this pull request needs your attention!

**Pull request closed without merging**